### PR TITLE
[KAFKA-6702]Wrong className in LoggerFactory.getLogger method

### PR DESCRIPTION
**Pull request by hejiefang**
[https://issues.apache.org/jira/browse/KAFKA-6702](https://issues.apache.org/jira/browse/KAFKA-6702)
Wrong className in LoggerFactory.getLogger method

### CODE REVIEW

File: streams/src/main/java/org/apache/kafka/streams/errors/LogAndContinueExceptionHandler.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
  * signals the processing pipeline to continue processing more records.
  */
 public class LogAndContinueExceptionHandler implements DeserializationExceptionHandler {
-    private static final Logger log = LoggerFactory.getLogger(StreamThread.class);
+    private static final Logger log = LoggerFactory.getLogger(LogAndContinueExceptionHandler.class);
```
Comment (omkreddy) [inline streams/src/main/java/org/apache/kafka/streams/errors/LogAndContinueExceptionHandler.java:32]: Pls fix checkstyle error due to unused StreamThread import

Comment (hejiefang) [reply streams/src/main/java/org/apache/kafka/streams/errors/LogAndContinueExceptionHandler.java:32]: I fix that，but another error appear. If there is any exception in test of testNoConsumeWithoutDescribeAclViaSubscribe, i have only change the class name of logger and do nothing for zookeeper.set.acl

Comment (omkreddy) [reply streams/src/main/java/org/apache/kafka/streams/errors/LogAndContinueExceptionHandler.java:32]: that may be transient error.. we can re-run the tests again by commenting "retest this please"

**Comment by omkreddy** [comment]
retest this please

File: streams/src/main/java/org/apache/kafka/streams/errors/LogAndContinueExceptionHandler.java (hunk 1 above)
Comment (hejiefang) [reply streams/src/main/java/org/apache/kafka/streams/errors/LogAndContinueExceptionHandler.java:32]: thanks

Comment (hejiefang) [reply streams/src/main/java/org/apache/kafka/streams/errors/LogAndContinueExceptionHandler.java:32]: Execuse me,is there any transient error in the test. I can not see the result now. It says 0 failed but JDK 7 and Scala 2.11 is FAILURE

**Comment by hachikuji** [comment]
Note `LogAndFailExceptionHandler` has the same issue. Maybe we should fix it as well?

**Pull request closed without merging**